tests/_test_msui/test_wms_control.py

```diff
@@ -162,15 +166,14 @@ def test_forward_backward_clicks(self):
             pass
 
     @pytest.mark.skip("Has a race condition where the abort might not happen fast enough")
-    def test_server_abort_getmap(self):
+    def test_server_abort_getmap(self, qtbot):
         """
         assert that an aborted getmap call does not change the displayed image
         """
-        self.query_server(self.url)
```

Comment:
It doesn't look quite right in this test. The order of the mouseClick and keyClick calls is reversed now. I think the keyClick should be in the with: block as well.